payara/Payara
# Fish 6495 - Hazelcast File Configuration in Payara Embedded

## Description
A customer would like to provide a path to a hazelcast configuration file when using Payara Embedded with Arquillian.

## Testing
### New tests
None

### Testing Performed
1. Download the attached reproducer application & hazelcast-config.xml
2. Modify the embedded arquillian configuration in arquillian.xml to specify a path to the hazelcast-text.xml file:
    <?xml version="1.0"?>
<arquillian xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
            xmlns="http://jboss.org/schema/arquillian"
            xsi:schemaLocation="http://jboss.org/schema/arquillian
                http://jboss.org/schema/arquillian/arquillian_1_0.xsd">

    <container qualifier="payara-embedded" default="true">
        <configuration>
            <property name="bindHttpPort">9990</property>
            <property name="hazelcastConfigurationFile">C:/Users/luise/Downloads/hazelcast-test.xml</property>
        </configuration>
    </container>
</arquillian>
3. Run the test with mvn clean install and the test should pass

### Testing Environment
Zulu JDK 1.8_212 on Windows 10 with Maven 3.8.4
